lightning-invoice/src/lib.rs

```diff
@@ -853,8 +853,8 @@ impl RawInvoice {
 		hash
 	}
 
-	/// Calculate the hash of the encoded `RawInvoice`
-	pub fn hash(&self) -> [u8; 32] {
+	/// Calculate the hash of the encoded `RawInvoice` which should be signed
+	pub fn signed_hash(&self) -> [u8; 32] {
```

Comment:
I find the naming `signed_hash` very confusing, since I read it as a hash that has *already* been signed. If `hash` is too ambiguous, maybe call it analogously `raw_hash`, or just `signing_hash`, `hash_to_sign`, or similar?

Reply:
How about [signable](https://www.merriam-webster.com/dictionary/signable)_hash?